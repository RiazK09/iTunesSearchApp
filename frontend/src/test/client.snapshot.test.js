import React from "react";
// Imported AddFavourites.js for testing.
import AddFavourites from '../components/AddFavourites'
// Imported react-test-renderer for rendering snapshots.
import renderer from "react-test-renderer";

/* Snapshot Test
• After renderer.create, I passed in the component that I want to render.
• I then called .toJSON() on it to get a JSON representation of the component. */

it("renders correctly", () => {
  const tree = renderer.create(<AddFavourites />).toJSON();
  expect(tree).toMatchSnapshot();
});