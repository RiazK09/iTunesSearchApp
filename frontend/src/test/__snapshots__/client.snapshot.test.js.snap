// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <span
    className="AddToFavouritesComponent"
  >
    Add to Favourites
  </span>
  <svg
    className="bi bi-heart-fill"
    fill="red"
    height="16"
    viewBox="0 0 16 16"
    width="16"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M8 1.314C12.438-3.248 23.534 4.735 8 15-7.534 4.736 3.562-3.248 8 1.314z"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;
